src/Compilers/CSharp/Test/Syntax/Parsing/ParserRegressionTests.cs

```diff
@@ -83,24 +81,7 @@ public void c024928()
         [Fact]
         public void TestBinary()
         {
-            // Apparently this fixed seed exposed a bug at some point
-            var random = new System.Random(12345);
-            // 40 million "character"s
-            const int n = 40 * 1000 * 1000;
-            var str = new string(' ', n);
-
-            unsafe
-            {
-                fixed(char* chars = str)
-                {
-                    for(int i = 0; i < n; i++)
-                    {
-                        chars[i] = (char)random.Next(char.MaxValue);
-                    }
-                }
-            }
-
-            var tree = CSharpSyntaxTree.ParseText(str);
+            CSharpSyntaxTree.ParseText(new RandomizedSourceText());
```

Comment:
Do we have some reason to believe that this modified test would have triggered the same (original) issue?

The (flakiness) failure I frequently see in this test is in the debug code inside the parser written to verify that the parsed text is consistent with the original; it creates a 40MB string to do that, which exhausts memory when tests are run in parallel. This doesn't really change that.